modules/date.js

```diff
@@ -0,0 +1,8 @@
+import { DateTime } from './luxon/src/luxon.js';
+
+export default function currentDate(container, main) {
+  const currentDate = document.createElement('p');
+  currentDate.className = 'date';
+  currentDate.textContent = DateTime.now().toFormat('MMMM dd, yyyy');
+  container.insertBefore(currentDate, main);
```

Comment:
- [ ] Let's use ES6 syntax for function creation, instead of the key work `function`, we can use `const currentDate = () =>` with arrow function.
- [ ] Let's do the same for the `function navController`